Hometask8/src/test/java/homework21/SelenideTest.java

```diff
@@ -0,0 +1,75 @@
+package homework21;
+
+import com.codeborne.selenide.*;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.*;
+import static org.testng.Assert.assertEquals;
+
+public class SelenideTest {
+    final String userName = "standard_user";
+    final String password = "secret_sauce";
+    final String inventoryPageTitle = "Products";
+    final String title = ".title";
+    final String cartBtnSelector = ".shopping_cart_link";
+    final String cartTitle = "Your Cart";
+    final String checkoutBtnId = "#checkout";
+    final String checkoutTitle = "Checkout: Your Information";
+    final int selectedItem = 2;
+    final String[] checkoutData = {"Kate", "Smith", "98765"};
+    final String overviewTitle = "Checkout: Overview";
+    final String productsInCartSelector = ".inventory_item_price";
+    final String summarySelector = ".summary_subtotal_label";
+    final String finishBtnId = "#finish";
+    final String completeTextSelector = ".complete-header";
+    final String completeText = "Thank you for your order!";
```

Comment:
test data and selectors should not be stored in one place as it makes mess, hard to find proper info then